controllers/url.js

```diff
@@ -10,7 +10,7 @@ const client = redis.createClient(REDISPORT);
 
 const baseURL = config.get('baseURL');
 
-async function shorten(req, res) {
+ exports.URLshorten= async (req, res)=> {
```

Comment:
Revert these changes and follow only the convention used in the project.

Reply:
revetred the changes